content/post/Android_Root101.en.md

```diff
@@ -1,31 +1,32 @@
 +++
 title = "A Guide to Flashing Custom ROMs [Android]"
-slug = "Flashing Custom ROMs 101"
+slug = "custom-rom-101"
```

Comment:
```suggestion
slug = "slapping-custom-roms-on-android-like-a-pro"
```
Just a suggestion :heart: 
And this was a joke, where I meant. Make descriptive slugs.

Reply:
👅 that slapping 😆